fb-negotiation/core/scenario.py

```diff
@@ -0,0 +1,43 @@
+from cocoa.core.scenario_db import Scenario as BaseScenario
+from cocoa.core.schema import Attribute
+from kb import KB
+
+class Scenario(BaseScenario):
+    # Agent ids
+    FIRST = 0
+    SECOND = 1
+
+    def __init__(self, uuid, book, hat, ball, attributes, kbs):
+        super(Scenario, self).__init__(uuid, attributes, kbs)
+        self.book_count = book
+        self.hat_count = hat
+        self.ball_count = ball
+        # self.bottom_line = 8
```

Comment:
The counts are already in the kb, so we can remove these in the scenario, i.e. the scenario just contains two kbs.